network/router/default.go

```diff
@@ -594,5 +609,5 @@ func (r *router) Stop() error {
 
 // String prints debugging information about router
 func (r *router) String() string {
-	return "router"
+	return "default router"
```

Comment:
This should either be called default or router. We only ever return single word strings for implementations to be stored in metadata.

Reply:
Yah, I hear ya, Maxi. This one escaped my attention somehow! I had a few routers running off different branches and this one got away. Nice spot!

Reply:
Come to think of it, I think we should return `"default"` and not `"router"` as we originally did. WDYT?

Reply:
Sounds good